docs/setup/manual_installation.md

```diff
@@ -0,0 +1,49 @@
+# Manual Installation
+
+## Install and setup
+You will need python 2.7.12+ and `virtualenvwrapper` installed to follow these instructions:
+
+```
+sudo apt-get install git python-dev python-pip
+sudo pip install virtualenv virtualenvwrapper --ignore-installed six
+```
+
+## Setup
+
+This should be run from your home directory:
+```
+source <(curl -s https://raw.githubusercontent.com/dimagi/commcare-cloud/master/control/init.sh)
+```
+
+Alternatively:
```

Comment:
I think there are many ways we could fetch and run `init.sh`, and I think it's OK to pick just one.

Personally, I'd go for the one above, because the path will be consistent with the `export PATH=$PATH:~/.commcare-cloud/bin` instruction below.